src/screens/base/screen.js

```diff
@@ -54,76 +54,87 @@ class Screen {
     }
     this._deviceInfo = _deviceInfo();
   }
+
   get Header() {
     return match.Label(this.screenText.Header);
   }
+
   get BackButton() {
     return match.accessible.BackButton(this.backLabel);
   }
+
   get OpenButton() {
     if (this.openBtn) {
       return match.accessible.Button(this.openBtn);
     }
     return match.Label(this.openLabel);
   }
+
   async Authenticate() {
     if (this._deviceInfo.useFaceID) {
       await device.matchFace();
     } else {
       await device.matchFinger();
     }
-    //HACK: the match can take some time so we need to wait
+    // HACK: the match can take some time so we need to wait
     await _sleep(5000);
   }
+
   async IsOn(buttonElement) {
     try {
-      await expect(buttonElement).toHaveValue("0");
+      await expect(buttonElement).toHaveValue('0');
       return false;
     } catch (err) {
       return true;
     }
   }
+
   async IsButtonOn(buttonElement) {
     return this.IsOn(buttonElement);
     // const buttonAttributes = await buttonElement.getAttributes();
     // return buttonAttributes.value == 1;
   }
+
   async SwipeUpUntilVisible(labelToSee) {
     await action.SwipeUpUntilVisible(labelToSee);
   }
+
   async SwipeDownUntilVisible(labelToSee) {
     await action.SwipeDownUntilVisible(labelToSee);
   }
+
   async CloseModal() {
     await action.SwipeDown();
   }
+
   async DismissAlert(label) {
     await this.Alert(label).tap();
   }
+
   Alert(buttonText) {
     return match.accessible.Button(buttonText);
   }
 }
 
-var _baseScreenTests = async function ({ openScreenFunc, skipClose = false }) {
-    let screen;
-    it("can open", async () => {
-      screen = await openScreenFunc();
-    });
-    it("has a Header", async () => {
-      await expect(screen.Header).toBeVisible();
+let _baseScreenTests = async function ({ openScreenFunc, skipClose = false }) {
+  let screen;
+  it('can open', async () => {
+    screen = await openScreenFunc();
+  });
+  it('has a Header', async () => {
+    await expect(screen.Header).toBeVisible();
+  });
+  it('has a Back Button', async () => {
+    await expect(screen.BackButton).toBeVisible();
+  });
+  if (!skipClose) {
+    it('can close', async () => {
+      await screen.BackButton.tap();
```

Comment:
I remember you talked about moving tests out of screens, but I cannot remember if you were planning to do all of them and if you were planning to do that now. So just pointing this out.

Reply:
yes I'm doing this slowly but surely, this file is actually already changed in my local, but it took quite a bit of finagling so i'll send that up as a seperate pull request.